scripts/installers/Dockerfile.ngramtools

```diff
@@ -0,0 +1,13 @@
+# How to use? Build it from NeMo root folder:
+# 1. git clone https://github.com/NVIDIA/NeMo.git && cd NeMo
+# 2. DOCKER_BUILDKIT=1 docker build -t nemo:22.12.2 -f ./scripts/installers/Dockerfile.ngramtools .
```

Comment:
Please add some docs on what this script does and when it should be used and how be called.